src/construction.cpp

```diff
@@ -1556,8 +1556,9 @@ int construction::adjusted_time() const
 }
 
 std::string construction::get_time_string() const
-{
-    const int turns = adjusted_time() / 100;
+{    
+    const float season_mult = calendar::turn.season_length()/91.0;
+    const int turns = (int)(season_mult * adjusted_time() / 100);
```

Comment:
My intent was that since we wanted (I thought) to scale most of the player's activities, including those like disassembly and mining, we could grab all the use cases at once right here, and not require tailored code all over the place.  If the pwoers that be settle on just construction, or just construction and crafting, then I agree, your suggestion would be the better fix, provided that experience gain also receives the same scaling to keep the XP per item constant.  Scaling crafting time is one thing, but scaling _grinding_ is not likely to make many friends.